apps/i18n/common/en_us.json

```diff
@@ -1863,6 +1863,28 @@
   "myProjects": "My Projects",
   "myProfessionalLearningCourses": "My Professional Learning Courses",
   "myWorkshops": "My Workshops",
+
+  "accountInformation_verified_teacher": "Verified Teacher",
+  "accountInformation_display_name": "Display Name",
+  "accountInformation_username": "Username",
+  "accountInformation_display_name_hint_teacher": "This is what your students see.",
+  "accountInformation_user_type": "Account type",
+  "accountInformation_email": "Email address",
+  "accountInformation_email_hint": "Make sure you can receive emails to this email address.",
+  "accountInformation_password": "Password",
+  "accountInformation_password_confirmation": "Password confirmation",
+  "accountInformation_current_password": "Confirm current password",
+  "accountInformation_current_password_hint": "We need your current password to confirm changes.",
+  "accountInformation_update_account_information": "Update account information",
+  "accountInformation_no_password_because_sponsored": "See your teacher to manage your password.",
+  "accountInformation_gender": "Gender (optional)",
+  "accountInformation_state": "State",
+  "accountInformation_update_field_parent_permission_required": "(You must obtain parental permission before updating this field)",
```

Comment:
Nit: Could the casing be consistently camel since the other strings follow this too?

Reply:
hah, I swear I thought I was following another convention I saw in this file... but now I can't find it 😅 

there's quite a mix of casing in that file, but I agree, better to keep it simple, I'll update these!

Reply:
I just saw this thread, I'll be commenting here too.
We prefer to use underscores to separate the hierarchy `"componentName_stringName"`.
For example it would be best to say `"accountInformation_updateAccountInformation"`